pkg/api/routes.go

```diff
@@ -482,26 +482,14 @@ func (rh *RouteHandler) GetReferrers(response http.ResponseWriter, request *http
 		return
 	}
 
-	// filter by artifact type
-	artifactType := ""
-
-	artifactTypes, ok := request.URL.Query()["artifactType"]
-	if ok {
-		if len(artifactTypes) != 1 {
-			rh.c.Log.Error().Msg("invalid artifact types")
-			response.WriteHeader(http.StatusBadRequest)
-
-			return
-		}
-
-		artifactType = artifactTypes[0]
-	}
+	// filter by artifact type (more than one can be specified)
+	artifactTypes := request.URL.Query()["artifactType"]
 
-	rh.c.Log.Info().Str("digest", digest.String()).Str("artifactType", artifactType).Msg("getting manifest")
+	rh.c.Log.Info().Str("digest", digest.String()).Interface("artifactType", artifactTypes).Msg("getting manifest")
 
 	imgStore := rh.getImageStore(name)
 
-	referrers, err := getReferrers(request.Context(), rh, imgStore, name, digest, artifactType)
+	referrers, err := getReferrers(request.Context(), rh, imgStore, name, digest, artifactTypes)
```

Comment:
Where is the artifactTypes variable cast to a list of strings? Or that is not necessary?

Reply:
Not, needed but agreed, definitely needs a test.